includes/Tracking.php

```diff
@@ -90,7 +108,38 @@ public function get_settings() {
 			'trackingAllowed' => $this->is_active(),
 			'trackingId'      => self::TRACKING_ID,
 			'trackingIdGA4'   => self::TRACKING_ID_GA4,
+			// This doesn't seem to be fully working for web properties.
+			// So we send it as both app_version and a user property.
+			// See https://support.google.com/analytics/answer/9268042.
 			'appVersion'      => WEBSTORIES_VERSION,
+			'userProperties'  => $this->get_user_properties(),
+		];
+	}
+
+	/**
+	 * Returns a list of user properties.
+	 *
+	 * @since 1.4.0
+	 *
+	 * @global string $wp_version WordPress version.
+	 *
+	 * @return array User properties.
+	 */
+	private function get_user_properties() {
+		global $wp_version;
+
+		$role        = ! empty( wp_get_current_user()->roles ) ? wp_get_current_user()->roles[0] : '';
+		$experiments = implode( ',', $this->experiments->get_enabled_experiments() );
+
+		return [
+			'siteLocale'         => get_locale(),
+			'userLocale'         => get_user_locale(),
+			'userRole'           => $role,
+			'enabledExperiments' => $experiments,
+			'wpVersion'          => $wp_version,
+			'phpVersion'         => PHP_VERSION,
+			'pluginVersion'      => WEBSTORIES_VERSION,
+			'numberOfUsers'      => is_multisite() ? get_user_count() : count_users()['total_users'],
```

Comment:
As maintainer of users in WordPress, can't recommend using `count_users`. It has a number of preference problems, related to the odd query that is run in this function. More detail in [#38741](https://core.trac.wordpress.org/ticket/38741). 

I can work on cached version for you if you want.

Reply:
Hmm interesting, thanks for sharing.

I think I'll just drop this. Not worth the hassle.